frameworks/FSharp/giraffe/src/App/Stock.fs

```diff
@@ -0,0 +1,42 @@
+﻿module Stock
+
+open Giraffe
+open Dapper
+open Npgsql
+open Microsoft.AspNetCore.Http
+open Models
+open FSharp.Control.Tasks
+open System.Text
+
+let application : HttpHandler = 
+    
+    let fortunes : HttpHandler = 
+        let extra = {id = 0; message = "Additional fortune added at request time."}
+        let extra' = Seq.singleton extra
+
+        fun (_ : HttpFunc) (ctx : HttpContext) ->
+            task {
+                use conn = new NpgsqlConnection(ConnectionString)
+                let! data = conn.QueryAsync<Fortune>("SELECT id, message FROM fortune")
+
+                let view = 
+                    data 
+                    |> Seq.append extra'
+                    |> Seq.sortBy (fun x -> x.message)
+                    |> HtmlViews.fortunes
+
+                // stock implementation does not allow to set content type for view rendering in 1.1.0
+                let bytes = 
+                    view 
+                    |> GiraffeViewEngine.renderHtmlDocument 
+                    |> Encoding.UTF8.GetBytes
+
+                ctx.SetContentType "text/html;charset=utf-8"
+                return! ctx.WriteBytesAsync bytes
+            }
+
+    GET >=> choose [

```

Comment:
The "GET" Method filter is more correct but adding unneeded bloat to the pipeline when it is not a test condition is just wasted resource, I would recommend leaving the method filter out to squeeze tiny bit more perf.

Reply:
thx for the spot, it should go